Resources/PO/LoginPage.robot

```diff
@@ -0,0 +1,34 @@
+*** Settings ***
+Library  SeleniumLibrary
+
+
+*** Variables ***
+${LOGIN_USERNAME_LOCATOR} =  //input[@id="user_email"]
+${LOGIN PASSWORD_LOCATOR} =  //input[@id="user_password"]
+${LOGIN_SUBMIT_LOCATOR} =  //input[@value="Submit"]
+${LOGIN_ERROR_TEXT_LOCATOR} =  //div[@class="panel-body"]
+
+
+*** Keywords ***
+Populate Login Form And Sumbit
+    [Arguments]    ${username}
+    ...            ${password}
+    Wait Until Page Contains Element  ${LOGIN_USERNAME_LOCATOR}
+    Input Text  ${LOGIN_USERNAME_LOCATOR}  ${username}
+    Input Password  ${LOGIN PASSWORD_LOCATOR}  ${password}
+    CLick Element  ${LOGIN_SUBMIT_LOCATOR}
+
+Verify Login Form Displayed
+    Wait Until Page Contains Element  ${LOGIN_USERNAME_LOCATOR}
```

Comment:
you duplicate this line in two different keywords. If element will change you will need to change it in multiple places: 

better to reuse this kw in `Populate Login Form And Sumbit`